app/controllers/api/v0/github_repos_controller.rb

```diff
@@ -3,9 +3,10 @@ module V0
     class GithubReposController < ApiController
       def index
         client = create_octokit_client
-        existing_user_repos = GithubRepo.
-          where(user_id: current_user.id, featured: true).pluck(:github_id_code) #=> [1,2,3]
-        existing_user_repos = Set.new(existing_user_repos)
+
+        existing_user_repos = current_user.github_repos.where(featured: true).
+          distinct.pluck(:github_id_code)
```

Comment:
`.distinct.pluck` tells the DB to select only unique values, so we don't have to use `Set.new` or `.to_set` in memory

Reply:
TIL! 🔥